act-rules-format/act-rules-format.bs

```diff
@@ -452,6 +480,10 @@ Note that input rules in a composite rule <em class="rfc2119">may</em> have diff
   </blockquote>
 </aside>
 
+#### Applicability Type Designation (optional)
```

Comment:
I'd like to see this concept added to glossary terms and expectations as well

Reply:
I will add this in a separate PR after this is merged.